lib/PipelineC/Tracing/Runner.cpp

```diff
@@ -489,6 +489,13 @@ llvm::Error Trace::run(const revng::tracing::RunTraceOptions Options) const {
       continue;
     }
 
+    TraceRunnerLogger << "Running #" << Command.ID << ": " << Command.Name;
+    for (const revng::tracing::Argument &Argument : Arguments) {
+      TraceRunnerLogger << ", " << Argument.toString();
+    }
+    TraceRunnerLogger << "\n";
+    TraceRunnerLogger.flush();
```

Comment:
1. Do not put in a logger `\n`, unless you want a multiline message.
2. Always terminate the message with `<< DoLog`.
3. Always wrap writing to a logger in a `if (Log.isEnabled())` (unless you use `revng_log`).

Reply:
1. From line 492 to line 495 it's a single long line, there's no newline. It is added on line 496
2. Ok
3. Isn't the point of the logger to do that on its own? If it's a performance concern I doubt the trace runner is really somewhere where the logging overhead matters.

Reply:
1. Do not put the newline. `<< DoLog` terminates the message for you and it's conceptually a newline.
2. .
3. It's most likely not a concern, but this is a policy and we do that everywhere.
   Let's not deviate from the policy unless there's a good reason.

Reply:
Fixed, added guard for enabled, and used `DoLog`